lib/ipmi_backend_utils.pm

```diff
@@ -67,7 +67,7 @@ sub get_dom0_serialdev {
         }
     }
     else {
-        $dom0_serialdev = 'ttyS1';
+        $dom0_serialdev = get_var("SERIALDEVORIG", "ttyS1");
```

Comment:
I do not think it is a good idea to introduce an undocumented variable here which looks like a workaround. I agree that 'ttyS1' should not be hardcoded when you look into aarch64 bare metal automation if this is what you want.

Why not just use the proper `SERIALDEV` here? It is not a problem of the backend to solve when other locations set a wrong value for SERIALDEV, right?

Reply:
Because the SERIALDEV is overwritten somehow during execution. At least, according to my observation, it is reset to 'sshserial' by some other subroutines.

Reply:
I fully agree with you that we can consider this bad design. But then this should be fixed properly, not by introducing the workaround. The example https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/Utils/Backends.pm#L50 found is the troublemaker so the problem why we have that needs to be understood and handled properly, e.g. discuss with the authors of that change.

Reply:
Changes have been made.